docs/src/developer/ide/tipps/index.md

```diff
@@ -0,0 +1,7 @@
+# IDE Docu
+
+![wrench](/wrench.png)
```

Comment:
because of vitepress configuration this is not necessary. all assets are stored and searched for in the `/public` directory so the only path that needs to be referenced is what comes after `/public`